src/azul/openapi/schema.py

```diff
@@ -192,12 +193,32 @@ def enum(*items: PrimitiveJSON, type_: TYPE = None) -> JSON:
             "bar"
         ]
     }
-    """
 
+    >>> assert_json(enum(items=['a']))
+    {
+        "type": "string",
+        "enum": [
+            "a"
+        ]
+    }
+    >>> assert_json(enum(*['a'], items=['b']))
+    Traceback (most recent call last):
+    ...
+    azul.RequirementError
+    """
+    if items is None:
+        # jsonschema validator package rejects tuples, so convert to list.
+        items_ = list(items_)
+    else:
+        require(len(items_) == 0)
+        # TODO: This isinstance check needs to be looser. allow mappings.
+        #    https://github.com/DataBiosphere/azul/issues/2375
```

Comment:
We don't commit commented out code. Can you check the C guide and reply with link to relevant section?

Reply:
https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#117disabling-sections-of-code